src/tabs/AOCTab/SolutionsView.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import SolutionView from './SolutionView';
+import { Dropdown, DropdownButton } from 'react-bootstrap';
+
+/**
+ * Generates the dropdown items for the AOC dropdown menu.
+ * 
+ * @param {number} numberOfItems The number of items we want to display
+ * @param {Function} setDay Hooks set function.
+ */
+const dropdownItems = (numberOfItems, setDay) => {
+  const items = [];
+  for (let i = 0; i < numberOfItems; i++) {
+    const day = i + 1;
+    items.push(
+      <Dropdown.Item onClick={() => {
+        setDay(day);
+      }}>
+        Day {day}
+      </Dropdown.Item>
+    );
+  }
+  return items;
+};

```

Comment:
pristine code, ur awesome

Reply:
LOL

Reply:
Can't tell if you forgot you wrote all of this, or if you're intentionally being funny

Reply:
it's the latter lmfao